test/timestamp.js

```diff
@@ -122,7 +115,11 @@ describe('timestamps', function() {
 
   it('convert to date', function() {
     return createInstance(
-               {timestampsInSnapshots: true}, DOCUMENT_WITH_TIMESTAMP)
+               {
+                 timestampsInSnapshots: true,
+                 keyFilename: './test/fake-certificate.json',
```

Comment:
Just curious: why is `keyFilename` passed here, but not in `'returned when enabled`?

Reply:
I meant to remove `keyFilename` from these callers and only set it in `createInstance()`. I removed it here and below.